tests/kuttl/patch-object.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -eo pipefail
+set -x
+
+
+sleep 3
```

Comment:
I'm open for a suggestion here. Unfortunately I could not find a solution with `kubectl wait` command for situations when resources does not exist yet. 

Insight: The CRDs are being created later then this script execution which means I get empty names for the CRDs.

Reply:
You said, you also need some timeout for this thing?

Maybe something like this would work:
```
for (( i=1; i<=5; i++ )); do sleep 1 && kubectl get objectuser; done
```

This should try 5 seconds to get objectuser and then return.

Reply:
Ok we went with the kuttle TestStep and used `sleep` :)